backend/app/services/chat/trigger/emitter.py

```
@@ -424,6 +453,155 @@ async def _update_execution_status(
+
+                if not subscription:
+                    logger.warning(
+                        f"[SubscriptionEmitter] Subscription {execution.subscription_id} not found for notification"
+                    )
+                    return
+
+                # Get subscription display name
+                from app.schemas.subscription import Subscription
+
+                subscription_crd = Subscription.model_validate(subscription.json)
+                subscription_name = (
```

Ruff (0.14.14)

[error] Local variable `subscription_name` is assigned to but never used

Remove assignment to unused variable `subscription_name`

(F841)